core/src/main/java/be/orbinson/aem/dictionarytranslator/servlets/action/ReplicateDictionaryServlet.java

```diff
@@ -46,8 +53,14 @@ protected void doPost(SlingHttpServletRequest request, @NotNull SlingHttpServlet
             Resource resource = resourceResolver.getResource(path);
 
             if (resource != null) {
-                DistributionRequest distributionRequest = new SimpleDistributionRequest(DistributionRequestType.ADD, true, path);
-                distributor.distribute("publish", resourceResolver, distributionRequest);
+//                DistributionRequest distributionRequest = new SimpleDistributionRequest(DistributionRequestType.ADD, true, path);
```

Comment:
Do not commit commented out code and never throw runtime exceptions, also remove logger without additional value or change message, format code